.changeset/olive-pugs-marry.md

```
@@ -0,0 +1,14 @@
+---
+'@c15t/backend': patch
+---
+
+Fix `www` handling in CORS origin matching
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)